jumpscale/clients/stellar/readme.md

```diff
@@ -154,3 +154,16 @@ JSX> j.clients.stellar.other_wallet.sign_multisig_transaction(tx_xdr)
 # last step is to remove the signer from the account
 JSX> j.clients.stellar.my_wallet.remove_signer(other_wallet.address)
 ```
+
+
+## Creating a sell order
+
+Here is an example of placing a selling order of XLM for TFT
+```python
+JS-NG> w = j.clients.stellar.get("w1")
+JS-NG> tft = w.get_asset_by_code("TFT")
+JS-NG> xlm = w.get_asset_by_code("XLM")
+JS-NG> w.place_selling_order(xlm, tft, "5", "20") # or w.place_selling_order(xlm, tft, decimal.Decimal(5), decimal.Decimal(10))
```

Comment:
please document what exactly this means, selling 5 xlm to receive  20 tft or sell 20 tft to receive 5 xlm